src/components/HOCs/withMap.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import TargetsMap from 'components/TargetsMap';
+
+const withMap = Component => () => (
```

Comment:
Maybe not a concern if this is for top level components only, but what about props?